cg/cli/clean.py

```diff
@@ -248,6 +250,32 @@ def hk_bundle_files(
     LOG.info(f"Process freed {round(size_cleaned * 0.0000000001, 2)}GB. Dry run: {dry_run}")
 
 
+@clean.command("flow-cells")
+@DRY_RUN
+def clean_flow_cells(context: CGConfig, dry_run: bool):
+    """Remove flow cells from the flow_cells and demultiplexed_runs folder."""
+
+    directories_to_check: List[Path] = []
+    for path in [Path(context.flow_cells_dir), Path(context.demultiplexed_flow_cells_dir)]:
+        directories_to_check.extend(get_directories_in_path(path))
+
+    raised_errors: List[bool] = []
+    for flow_cell_directory in directories_to_check:
+        clean_flow_cell_api = CleanFlowCellAPI(
+            flow_cell_path=flow_cell_directory,
+            status_db=context.status_db,
+            housekeeper_api=context.housekeeper_api,
+            dry_run=dry_run,
+        )
+        try:
+            raised_errors.extend(clean_flow_cell_api.delete_flow_cell_directory())
+        except Exception as error:
+            LOG.error(str(error))
+            continue
+    if any(raised_errors):
+        click.Abort
```

Comment:
I tried to implement a similar thing to as we did in `finish flow-cell-all` so that it tries the cleaning for all and aborts if anything went wrong, but after it is done with everything... I will need to test this if it works as intended.

Reply:
Yes I am working this part out today a bit more clearly with some testing, very much an afternoon-brain implementation currently.

Reply:
I implemented a similar approach